permits/templates/permits/permit_request_detail.html

```diff
@@ -35,37 +35,48 @@ <h1>
     <div class="row floating-elem">
       <div class="col">
         <dl>
-        <dd>
-          <h4>{% trans "Auteur-e" %}</h4>
-        </dd>
-        <dd>
-          <a href="mailto:{{ permit_request.author }} <{{ permit_request.author.user.email }}>" target="_blank" rel="noreferrer">
-            {{ permit_request.author.user.first_name }} {{ permit_request.author.user.last_name }} &lt;{{ permit_request.author.user.email }}&gt;
-          </a>
-        <dd>
-          {{ permit_request.author.address }} {{ permit_request.author.zipcode }} {{ permit_request.author.city }}
-        </dd>
-        <dd>  
-          {{ permit_request.author.phone_first }}  {{ permit_request.author.phone_second }}
-        </dd>
-        <dd>
-          {{ permit_request.author.company_name }} {{ permit_request.author.vat_number }}
-        </dd>
-        <dd>
-          {% if permit_request.validation_pdf %}
           <dd>
-            {% trans "Document délivré :" %}
+            <h4>{% trans "Auteur-e" %}</h4>
           </dd>
           <dd>
-              <i class="fa fa-download" aria-hidden="true"></i>
-              <a href="{{ permit_request.validation_pdf.url }}" target="_blank" rel="noreferrer">
-                Télécharger le document PDF
-              </a>
+            <a href="mailto:{{ permit_request.author }} <{{ permit_request.author.user.email }}>" target="_blank" rel="noreferrer">
+              {{ permit_request.author.user.first_name }} {{ permit_request.author.user.last_name }} &lt;{{ permit_request.author.user.email }}&gt;
+            </a>
+          </dd>
+          <dd>
+            {{ permit_request.author.address }}, {{ permit_request.author.zipcode }} {{ permit_request.author.city }}
+          </dd>
+          <dd>
+            {% if permit_request.author.phone_first %}
```

Comment:
`phone_first `first it not nullable in model, thus this `if `does not make sens IMHO: https://github.com/yverdon/geocity/blob/f2d08821601583c6809a57ae7cf2ada52841bd33/permits/models.py#L259-L268

Reply:
Ok, you're right (if it'll stay like that forever).